rotkehlchen/db/evmtx.py

```diff
@@ -35,7 +35,8 @@
 
 TRANSACTIONS_MISSING_DECODING_QUERY = (
     'evmtx_receipts AS A LEFT OUTER JOIN evm_tx_mappings AS B ON A.tx_hash=B.tx_hash '
-    'AND A.chain_id=B.chain_ID WHERE B.tx_hash is NULL'
+    'AND A.chain_id=B.chain_ID LEFT JOIN evm_transactions AS C on '
+    'A.tx_hash=C.tx_hash WHERE B.tx_hash is NULL'
```

Comment:
Hmmmm I am a bit confused here. What sense does `WHERE B.tx_hash is NULL` make if it `evm_tx_mappings.tx_hash` can never be null? https://github.com/rotki/rotki/blob/1c29fb705f38c097d9da4f88a4ef98725cb80e55/rotkehlchen/db/schema.py#L530

Maybe the check should be then that `evmtx_receipts.tx_hash` is not in `evm_tx_mappings.tx_hash` ?

Reply:
the `tx_hash` gets null when you do the join. in the table it can't be null but it is when you do the join that you the filtering